Source/WebCore/html/shadow/attachmentElementShadow.css

```diff
@@ -76,8 +99,13 @@ div#attachment-information-area {
 div#attachment-information-block {
     grid-row: 1;
     grid-column: 1;
+#if (defined(WTF_PLATFORM_IOS_FAMILY) && WTF_PLATFORM_IOS_FAMILY)
+    margin-left: 6px;
+    margin-right: 26px;
```

Comment:
I missed this in earlier reviews, but this should probably use `margin-inline-start` / `margin-inline-end`, in order to support RTL languages.

I would recommend testing RTL, and then a followup change, since there are probably other places that need to change too.

Reply:
Follow-up: rdar://106966312 (Wide-layout attachment: Verify that RTL display is correct)